virtcontainers/utils/utils_linux_test.go

```diff
@@ -36,16 +39,27 @@ func TestFindContextID(t *testing.T) {
 
 func TestGetDevicePathAndFsTypeEmptyMount(t *testing.T) {
 	assert := assert.New(t)
-	_, _, err := GetDevicePathAndFsType("")
+	_, _, _, err := GetDevicePathAndFsTypeOptions("")
 	assert.Error(err)
 }
 
 func TestGetDevicePathAndFsTypeSuccessful(t *testing.T) {
 	assert := assert.New(t)
 
-	path, fstype, err := GetDevicePathAndFsType("/proc")
+	cmdStr := "grep '/proc '  /proc/mounts "
```

Comment:
It would be safer to anchor this using something like:

```suggestion
	cmdStr := "grep ^proc /proc/mounts"
```

An alternative to the `exec` + `grep` dance is to use `ioutil.ReadFile()`.

Reply:
Let me know if you prefer ReadFile. For now I have kept the `exec + grep` combination with your suggested changes.